pkgs/by-name/go/goat/package.nix

```diff
@@ -0,0 +1,29 @@
+{ lib
+, fetchFromGitHub
+, buildGoModule
+,
+}:
+buildGoModule rec {
+  pname = "goat";
+  # Upstream currently isn't doing tags/releases, so using the Go pseudo-versioning instead.
+  # See https://pkg.go.dev/github.com/blampe/goat?tab=versions.
+  version = "v0.0.0-20220815015552-07bb911fe310";
```

Comment:
Please use the standard versioning scheme 
`unstable-2022-08-15`

Reply:
I this documented somewhere? I'm always a bit lost where to find these standards.

Thanks for reviewing!

Reply:
https://nixos.org/manual/nixpkgs/stable/#sec-package-naming